packages-exp/auth-exp/src/core/auth/firebase_internal.test.ts

```diff
@@ -70,13 +70,9 @@ describe('src/core/auth/firebase_internal', () => {
     beforeEach(async () => {
       user = testUser(auth, 'uid', undefined, true);
       await auth.updateCurrentUser(user);
-      sinon.stub(user.stsTokenManager, 'getToken').returns(
```

Comment:
this causes the `gets called on subsequent updates` test case to fail now... two options:

- change the check that it only calls it once (for new token, but not again for same token)
- parameterize the stub to generate a random token each time (or keep track of index like access-token-1, access-token-2, etc)

thoughts?

Reply:
I like #2. Should be easy with callsFake

Reply:
done